msu/squirrel_hooks/mod_hooks.nut

```diff
@@ -4,6 +4,7 @@
 	function add( _string )
 	{
 		::MSU.requireString(_string);
+		::logError(_string);
```

Comment:
Is this intended?

Reply:
Yes, if this doesn't print here it might not print at all if there are script errors in the code.